packages/logs/src/transport/startLoggerBatch.ts

```diff
@@ -0,0 +1,27 @@
+import { Batch, Configuration, Context, HttpRequest, EndpointBuilder } from '@datadog/browser-core'
+
+export function startLoggerBatch(configuration: Configuration) {
+  const primaryBatch = createLoggerBatch(configuration.logsEndpointBuilder)
+
+  let replicaBatch: Batch | undefined
+  if (configuration.replica !== undefined) {
+    replicaBatch = createLoggerBatch(configuration.replica.logsEndpointBuilder)
+  }
+
+  function createLoggerBatch(endpointBuilder: EndpointBuilder) {
+    return new Batch(
+      new HttpRequest(endpointBuilder, configuration.batchBytesLimit),
+      configuration.maxBatchSize,
+      configuration.batchBytesLimit,
+      configuration.maxMessageSize,
+      configuration.flushTimeout
+    )
+  }
+
+  return (message: Context) => {
```

Comment:
NIT: what about keeping an object with an `add` method, and do the same for returned value of `startLoggerEventBridge`? It could be a bit more explicit.

Reply:
So you prefer` transport.add(message)` instead of `transport(message)`. I suggest you look at the conversation with Bastien: https://github.com/DataDog/browser-sdk/pull/1155#discussion_r745408409